cutplace/interface.py

```diff
@@ -524,9 +523,28 @@ def as_sql_create_table(self, dialect='ansi'):
             if len(constraint) > 0:
                 constraints += constraint + ",\n"
 
+        constraints = constraints.rsplit(',', 1)[0]
+
         create_table += constraints
 
-        create_table += ");"
+        create_table += "\n);"
+
+        con = None
```

Comment:
Avoid (terse) abbreviations. As we are connecting to a database anyway, better use `temp_database`